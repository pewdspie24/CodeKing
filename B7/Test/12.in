53
9 1613347275
680563384 330696419
803187167 882217697
177206733 882171924
765002300 130087343
358746067 769788864
74811737 801401191
997283985 493754771
92640219 699380789
565789965 535742956
6 495770568
43369491 172088662
263334791 938265943
128618113 378895956
126855728 548787294
596555353 719381889
297127023 19187678
1 3100640603
835940044 725397665
6 3181112781
417640105 878743375
180449195 227490672
89085784 384630121
571722859 293675256
182313487 465397760
309274238 638308477
8 215480966
262806047 511471700
682230932 1265367
884450565 173786700
874374582 466434561
169055235 817817138
567358403 457713748
407040501 537256984
382571925 369353931
6 1123008437
582410663 271195724
190112566 568200654
178901704 230202163
59860995 681269325
456901527 246187069
77931099 893686984
1 2838461115
528589046 316516954
10 3190639306
587571875 857099397
228388185 231204255
527754373 628984054
266188898 47908483
21820875 123670858
2162188 217829344
906760443 88781958
699267763 515243854
855749701 400807563
48061090 732630080
10 2567494895
759368562 704149947
215665007 310093651
56963563 256657102
121300160 323875946
59799177 53671445
676066579 854796949
641047561 35984560
460005570 257824758
257221069 220264550
485630507 545646579
6 3262628370
699264778 604292582
5108807 971405612
400165170 332828273
731441374 158187850
287003105 492083618
9454244 538879479
5 2456997229
72878464 225485313
490966460 868189176
757090937 609984745
957726573 193547438
708595976 856421958
10 4042594537
604615284 343367046
661277942 639688776
127949663 208408320
286509798 411174870
876331903 595770515
521987054 976105338
68238510 464545969
165183093 251049638
646415149 867024705
16833522 650615385
8 3835592373
389012784 16768669
874197696 105425298
224844790 794628248
18380690 258300725
162810616 222217630
596007108 294973016
612652224 667590640
526602517 226717316
8 4185719809
572854907 778232209
360236756 247500648
574087870 609599907
795596044 937586635
138344571 157615609
512255256 592146433
165509643 7389864
791355167 872694023
2 27334395
227143591 961672082
48752046 759924302
7 785849667
641764548 84663380
177094065 617309220
162643243 887486795
808960129 115568415
559220075 576269558
980545642 228178013
490691009 581960928
2 1625847511
173529368 765089744
934576811 179297966
8 3923848674
155665779 847249571
322864936 686349192
561945061 869276993
772790444 619472082
809518018 156573323
663621425 145876865
52521552 282300748
256151240 137649914
4 3696498284
819950007 73689032
333876563 953364967
904753355 326688939
28250521 28552745
9 446086576
342510439 247985986
863879330 538989745
230445300 108967171
257343927 190919392
788792422 9261277
951249699 527413303
406431777 651420301
39163175 240032179
326119634 493820166
7 2854127599
620479642 497351455
798955791 742439729
627345920 230426913
592797366 596634299
509581743 833687236
22940376 92522710
602011981 897992495
6 1658650971
376399306 248051004
151217737 724547304
247084959 122920584
238409554 136525019
404419819 954727937
208753330 412177170
7 3398140132
777869242 945694554
639451917 268984483
534844852 150897492
262451342 84861704
261406139 121565054
341881317 319156242
334412277 748742986
7 3026737636
691300879 816332723
442189727 905552233
408000613 868693100
647065901 639268446
738360055 121841768
165821558 510991315
466715666 71593642
8 2672366163
682626326 878321417
439775016 767544059
99932847 372156923
136912789 423535419
566261582 977735524
147542167 809416122
831889251 743808959
562245542 635279754
7 2282087252
148078258 78856260
847624203 46146848
307523328 245964919
601326331 155219635
9021085 941358863
213598885 259212655
188557878 599413240
9 1991136260
53295196 890019279
636919377 589387148
402638235 169736668
731557065 73259980
125242428 807155226
777652314 864497084
130061724 188485286
204736927 956922473
998223644 845890172
2 3862938624
95713170 940740024
129183554 504449820
3 2517343769
34350605 144768704
878706825 493864977
406772899 996514870
3 2410722638
920296165 627075831
741687672 905609960
292701040 525815806
9 2596245810
281163554 84502882
485212302 909180637
456253353 447124844
274442771 830223334
593067391 591517745
412374585 223227107
109443773 357985360
550440818 452360170
102165160 716538669
8 1658180621
867152099 665917225
411730130 676385168
828814696 101089587
836695456 741050234
693722327 49680223
416582770 30710966
931374883 911605142
842974994 792893436
10 3492544887
996784138 668860034
839336845 940352321
340117035 798437369
705115064 243434271
57762035 931453151
871909156 388717300
957329734 797461209
803918095 420486359
355966686 459395335
715133188 318908230
9 1464895342
797534952 893647899
502505733 99104821
949103613 518219999
499876380 95659400
461283895 293490596
777368650 494909561
465958382 898702258
618784646 966746657
638292872 382694913
8 867887772
237585656 664274098
449242178 528643631
247279907 703239346
12158626 456561644
711925490 429463039
100461394 897579153
432635763 812475530
234910318 571834075
2 4057279837
922350868 163105239
344636776 415585587
7 2914917355
743571124 566107425
109889405 890633744
351904233 596359334
365983970 369095533
528405379 848083703
893772776 152708481
278680070 171853058
10 3961522557
894897229 511946113
693816612 245689709
668195613 167342833
698229037 88181315
444800977 225938923
989644773 735955949
291615758 86752998
700959560 735150707
695570470 790995041
418748688 436598530
4 2302898007
782799315 120057353
942621955 647623790
388181172 72080422
6763408 631101552
3 3362258578
569047672 180456
158694608 848369764
28832399 26700377
1 2682041628
217770558 764465559
6 245871044
257979062 672075551
299358887 289024590
206758806 607436923
970934391 957088112
551407746 902582741
869723535 815666760
9 3476691994
63509581 917249437
578602913 324003522
175429977 134299791
50580435 439818416
532302392 546638985
924895811 303840658
167029451 945712059
371402640 861243676
731653586 432648994
9 714389043
116109834 783882827
110571647 900010451
157154013 964941728
230670894 278803953
823192638 439012063
363065345 263460572
913943206 282613334
82538434 195878035
15374638 549765762
1 207828642
527650619 380340984
10 2474975573
320316385 863790961
134098789 910922393
1474854 581767433
900764797 614169308
983117878 579206531
527313689 18494338
929019778 560426628
480064125 227448344
802802614 199588631
347520139 192948159
9 80270370
676264065 314801551
791682918 905123120
851633067 104091800
297019813 36193073
923926711 430739699
106696095 678170879
413759092 747327039
903697373 978689723
95517083 940588372
7 3494356878
456901652 338370008
803993709 36073030
249775606 814710923
752992048 665288076
600888852 354728266
523039248 768647708
736773337 150569662
9 619184790
878201473 479918302
962584414 187724776
164904387 681949236
207875129 234492888
539719088 848259871
353246744 728538601
508842941 79972746
264289794 112956982
12489502 601458757
7 2285249906
536713285 987087218
172029379 388185572
583486960 667298159
108561936 872351535
524281756 81201482
639795408 191578280
337588133 94921795
6 2462666236
712416142 111700149
193919307 119973600
530275292 952995528
240504876 844862217
212209273 208198213
606170191 795900611
9 2788667477
645315033 941514808
473520237 771704845
333582379 264113426
633327424 958527192
172666696 18739838
13168982 908190187
39991222 96288154
632426693 624814622
694459244 582009079
2 3309990019
895235226 790689884
482243545 918410602
