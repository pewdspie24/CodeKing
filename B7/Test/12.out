2
1
22
11
1
3
12
4
4
6
3
13
5
23
1
1
2
6
5
1
7
11
8
5
22
3
3
MISSION FAILED!
6
7
12
2
7
3
2
6
4
6
4
6
MISSION FAILED!
1
6
1
1
4
1
5
1
5
4
15
25
